tutorials/pl/basic-snap-usage-pl/basic-snap-usage.md

```diff
@@ -0,0 +1,329 @@
+categories: packaging
+language: pl
+status: Published
+feedback link: https://github.com/ubuntu/codelabs/issues
+tags: snap, usage, beginner, idf-2016, polish
+difficulty: 1
+translator: Marcin Mikołajczak
+published: 2017-11-28
+
+---
+
+# Podstawy korzystania ze snapów
+
+## Wprowadzenie
+Duration: 1:00
+
+: Switch language: PL | [EN](tutorial/basic-snap-usage)
+
+Witamy w świecie snapów! W tym poradniku opisane zostaną podstawy korzystania z pakietów [snap] na Twojej dystrybucji Linuxa oraz poznasz korzyści wynikające z ich działania.
+
+
+![IMAGE](./snaps-hero.png)
+
+
+### Czego się nauczysz?
+
+  - Jak zainstalować usługę snapd na Twoim systemie.
+  - Jak szukać oprogramowania.
+  - Jak sprawdzić listę zainstalowanego oprogramowania.
+  - Jak aktualizowane są snapy.
+  - Czym są kanały snap i jak korzystać z nich.
+  - Jak wrócić do poprzedniej wersji snapa.
+
+Jeżeli wydaje Ci się to natłokiem informacji, przekonasz się, że jest inaczej. To wszystko jest bardzo proste dzięki prostocie interfejsu wiersza poleceń snapa.
+
+### Czego potrzebujesz?
+
+  - Dowolnej wspieranej dystrybucji systemu GNU/Linux (przeczytaj następny krok, aby poznać szczegóły).
+  - Podstawowej wiedzy dotyczącej korzystania z wiersza poleceń.
+
+Ten poradnik jest skupiony na poleceniu snap. Możesz po prostu skopiować i wkleić te polecenia.
+
+Survey
+: Jak wykorzystasz ten poradnik?
+ - Tylko przeczytam go
+ - Przeczytam i wykonam ćwiczenia
+: Jakie jest Twoje obecne doświadczenie?
+ - Nowicjusz
+ - Średniozaawansowany
+ - Doświadczony
+
+## Konfiguracja
+Duration: 2:00
+
+### Instalacja snapd
+`snapd` jest usługą, która działa na Twoim urządzeniu i monitoruje zainstalowane snapy, współdziała ze sklepem i dostarcza polecenie snap z którego będziesz korzystać. Instalacja na wymienionych poniżej dystrybucjach jest bezproblemowa.
+
+
+![IMAGE](./hero-artwork.png)
+
+
+
+### Różne dystrybucje:
+
+**Arch**
+
+```bash
+$ sudo pacman -S snapd
+
+# aktywuj usługę snapd:
+sudo systemctl enable --now snapd.socket
+```
+
+**Debian (tylko Sid)**
+
+```bash
+$ sudo apt install snapd
+```
+
+**Fedora**
+
+```bash
+$ sudo dnf copr enable zyga/snapcore
+$ sudo dnf install snapd
+
+# aktywuj usługę snapd:
+$ sudo systemctl enable --now snapd.service
+
+# wsparcie SELinux jest w fazie beta, obecnie:
+$ sudo setenforce 0
+
+# aby zachować to ustawienie, edytuj plik /etc/selinux/config, dodaj ustawienia SELINUX=permissive i uruchom system ponownie.
+```
+
+**Gentoo**
+
+Zainstaluj [gentoo-snappy overlay].
+OpenEmbedded/Yocto
+Zainstaluj [snap meta layer].
+
+**openSuSE**
+
+```bash
+$ sudo zypper addrepo http://download.opensuse.org/repositories/system:/snappy/openSUSE_Leap_42.2/ snappy
+$ sudo zypper install snapd
+```
+
+**OpenWrt**
+
+Aktywuj [snap-openwrt feed].
+
+**Ubuntu**
+W wersji 16.04 LTS desktop wszystko powinno być domyślnie aktywne. Na 14.04, wykonaj instalację poleceniem:
+
+```bash
+$ sudo apt install snapd
```

Comment:
You have to ask the user to reboot after installing snapd because that will pull in a new kernel as a dependency.